src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1774,7 +1775,7 @@ public function get_price_html( $product, $attendee = false ) {
 		}
 
 		$price = get_post_meta( $product_id, '_price', true );
-		$price = tribe_format_currency( $price, $product_id );
+		$price = Tribe__Tickets__Commerce__Currency::get_instance()->format_currency( $price, $product_id );
```

Comment:
since you registered `tickets.commerce.paypal.currency` in the container remove the `Tribe__Tickets__Commerce__Currency::get_instance()` method and call, instead, `tribe( 'tickets.commerce.paypal.currency' )` anywhere you would call `Tribe__Tickets__Commerce__Currency::get_instance()`

Reply:
the purpose of the container, here used as a service locator, is to avoid classes from knowing or caring about being used as singletons